backend/tracim_backend/command/search.py

```diff
@@ -28,10 +28,10 @@ def _index_all_contents(self) -> None:
         nb_index_errors = index_contents_result.get_nb_index_errors()
         nb_contents_to_index = index_contents_result.get_nb_contents_to_index()
         if nb_index_errors == 0:
-            print("All {} content where indexed".format(nb_contents_to_index))
+            print("All {} content(s) were indexed".format(nb_contents_to_index))
```

Comment:
I would expect All to always be followed by a plural, and all 1 content look a bit weird.

```suggestion
            print("{} content(s) were indexed".format(nb_contents_to_index))
```